src/cmap/connection_pool_events.ts

```diff
@@ -187,9 +187,18 @@ export class ConnectionPoolClearedEvent extends ConnectionPoolMonitoringEvent {
   /** @internal */
   serviceId?: ObjectId;
 
+  interruptInUseConnections?: boolean;
+
   /** @internal */
-  constructor(pool: ConnectionPool, serviceId?: ObjectId) {
+  constructor(
+    pool: ConnectionPool,
+    {
+      serviceId,
+      interruptInUseConnections
+    }: { serviceId?: ObjectId; interruptInUseConnections?: boolean } = {}
```

Comment:
I'm open to refactoring this if people want.  This is the only command monitoring event that now takes an object instead of standalone parameters.  I chose this approach even though it deviates from our convention because this approach allows us not to worry about optional argument handling.

Reply:
We do have another CMAP event that does pass an object: https://github.com/mongodb/node-mongodb-native/blob/d989f44b0b6089ad7e9389997e34dc005392c69e/src/cmap/connection_pool_events.ts#L74 - so my preference would be to at least keep them in line which ever way we choose since one is a named object and the other is not.